pipelines.yml

```diff
@@ -59,6 +59,7 @@ pipelines:
           <<: *UPDATE_COMMIT_STATUS
           onExecute:
             - cd $res_jfrogCliGit_resourcePath
+            - npm install -g npm
```

Comment:
Do we actually to install npm with npm?
If we do, let's add a comment explaining the reason we do this.

If we intended to install yarn here, I suggest we do not install it during the build, but instead add yarn to https://github.com/jfrog/jfrog-ecosystem-integration-env which is the container used for by this pipelines.yml. We actually need yarn to be included in this image anyway, because it is the base image for jfrog-cli's full docker image.